src/pages/assets.tsx

```diff
@@ -116,7 +143,7 @@ const AssetsPage = ({ data }) => {
           <AssetDownload
             title={translateMessageId("page-assets-hero", intl)}
             alt={translateMessageId("page-assets-hero", intl)}
-            image={data.hero}
+            image={getImage(data.hero)}
```

Comment:
I think we don't need to do `getImage` here since `AssetDownload` is already doing that for us.